### PR TITLE
fixed the bug of scrolling the screen

### PR DESCRIPTION
**Pull request by PranshuPandya**
I have fixed the bug of scrolling the screen on MyOrderPage.dart

## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

Basically the problem was that container was used in the SingleChildScrollView, while container has fixed hieght so it is unscrollable so i have added listview builder insted of container to make screen viewable.

## How Has This Been Tested?

I have tested on my device motorolla fusion plus and it is working fine.

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

Note: my vs code has auto formater and thus code on the 3 pages ahas got formated in standard style i am very sorry for that but i have expalined by change above so you can just check that and ignore other due to formating.

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g.never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)
 WhatsApp Image 2021-10-02 at 00 16 41

WhatsApp Image 2021-10-02 at 00 16 41 (1)

### CODE REVIEW

**Comment by Akshay0701** [comment]
there are some in this branch

**Comment by Akshay0701** [comment]
there is some conflicts in this branch

**Comment by PranshuPandya** [comment]
I have resolved the confliict.

File: lib/screens/homepage.dart [hunk 1]
```diff
@@ -348,4 +348,3 @@ class _HomePageContentState extends State<HomePageContent> {
     );
   }
 }
-
```
Comment (Akshay0701) [inline lib/screens/homepage.dart:351]: could you please undo this

File: lib/widgets/orderwidget.dart [hunk 2]
```diff
@@ -71,14 +99,16 @@ class _OrderWidgetState extends State<OrderWidget> {
         type: StepperType.horizontal,
         physics: NeverScrollableScrollPhysics(),
         controlsBuilder: (BuildContext context,
-            {VoidCallback onStepContinue, VoidCallback onStepCancel}) =>
-            Container(height: 0.0,),
+                {VoidCallback onStepContinue, VoidCallback onStepCancel}) =>
+            Container(
+          height: 0.0,
+        ),
```
Comment (Akshay0701) [inline lib/widgets/orderwidget.dart:105]: can you explain why did you change this?

File: lib/widgets/orderwidget.dart [hunk 3]
```diff
@@ -40,19 +40,47 @@ class _OrderWidgetState extends State<OrderWidget> {
...
+            trailing: Text(
+              widget.request.total + " Rs.",
+              style: TextStyle(
+                color: UniversalVariables.orangeColor,
+                fontSize: 20.0,
+                fontWeight: FontWeight.bold,
+              ),
+            ),
           ),
-
```
Comment (Akshay0701) [inline lib/widgets/orderwidget.dart:48]: revert this line

File: lib/widgets/orderwidget.dart [hunk 4]
```diff
@@ -6,7 +6,7 @@ import 'package:food_delivery_app/utils/universal_variables.dart';
 import 'foodTitleWidget.dart';
 
 class OrderWidget extends StatefulWidget {
-  Request request;
+  final Request request;
```
Comment (Akshay0701) [inline lib/widgets/orderwidget.dart:9]: any reason for keeping it final?

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
Thank you! @PranshuPandya for your contribution,

i mentioned few questions apart from that LGTM

File: lib/widgets/orderwidget.dart (hunk 4 above)
Comment (PranshuPandya) [reply lib/widgets/orderwidget.dart:9]: yeah it is unmutable. It was showing some warning so i added final

File: lib/screens/homepage.dart (hunk 1 above)
Comment (PranshuPandya) [reply lib/screens/homepage.dart:351]: yeah done

File: lib/widgets/orderwidget.dart (hunk 2 above)
Comment (PranshuPandya) [reply lib/widgets/orderwidget.dart]: it was changed unintentionally due to auto formater of vs code I have made it as it is of master now.

File: lib/widgets/orderwidget.dart (hunk 3 above)
Comment (PranshuPandya) [reply lib/widgets/orderwidget.dart]: yeah

File: lib/widgets/orderwidget.dart (hunk 2 above)
Comment (Akshay0701) [reply lib/widgets/orderwidget.dart]: great

**Review by Akshay0701 — APPROVED** [review]
Removed few comments, looks good to go now, 

Thanks @PranshuPandya for these contributions

merging now, cheers
